garage/envs/mujoco/gather/gather_env.py

```diff
@@ -27,66 +28,87 @@
 
 APPLE = 0
 BOMB = 1
+PyMjrRect = cymj.PyMjrRect
 
 
 class GatherViewer(MjViewer):
     def __init__(self, env):
         self.env = env
-        super(GatherViewer, self).__init__()
+        super(GatherViewer, self).__init__(self.env.wrapped_env.sim)
         green_ball_model = load_model_from_path(
             osp.abspath(osp.join(MODEL_DIR, 'green_ball.xml')))
         self.green_ball_renderer = EmbeddedViewer()
         self.green_ball_model = green_ball_model
+        self.green_ball_sim = MjSim(self.green_ball_model)
         self.green_ball_renderer.set_model(green_ball_model)
         red_ball_model = load_model_from_path(
             osp.abspath(osp.join(MODEL_DIR, 'red_ball.xml')))
         self.red_ball_renderer = EmbeddedViewer()
         self.red_ball_model = red_ball_model
+        self.red_ball_sim = MjSim(self.red_ball_model)
         self.red_ball_renderer.set_model(red_ball_model)
 
     def start(self):
-        super(GatherViewer, self).start()
+        # super(GatherViewer, self).start()
```

Comment:
Is there a reason to comment it out rather than simply deleting this line?

Reply:
There is no start() function in MjViewer.

Reply:
I would just remove the comment.